apps/admin-web/src/app/components/RouteItem.tsx

```diff
@@ -114,37 +114,35 @@ const RouteItem = ({ route, selectedItem, setSelectedItem }: Props) => {
         {route?.titleIcons?.map((icon) => (
           <GetLinkableIcon {...icon} />
         ))}
-        {selectedItem === route.id && (
-          <Button
-            variant="contained"
-            size="small"
-            sx={{
-              mr: 2,
-              backgroundColor: getColors().textColor,
-              maxHeight: 30,
-              alignSelf: 'center',
-            }}
-            onClick={() => {
-              if (route.method?.toUpperCase() === 'GET') {
-                openInNewTab(route.path.toString());
-              } else {
-                // If not a POST, currently assumes response is always JSON, fetch via API then open
-                fetch(route.path.toString(), {
-                  method: route.method,
-                  headers: {
-                    Accept: 'application/json',
-                    'Content-Type': 'application/json',
-                  },
-                  body: route.method === 'GET' ? undefined : '{}',
-                })
-                  .then((r) => r.json())
-                  .then(openJsonInNewTab);
-              }
-            }}
-          >
-            <OpenInNew />
-          </Button>
-        )}
+        <Button
```

Comment:
I get the change because you want to be able to access the details of the variant. Now that we show the variant info when collapsed its reasonable to have it visible. I do still doubt though the frequency of use when the user has the route collapsed. As as user I would probably be clicked into a route to compare the responses of different variants of the same route. Lets get a few opinions on this before we commit to it.

Reply:
As a tester I have to click multiple extra times 😢 

I think UI looks fine either way, +1 from me for always leaving this link available.